src/App.js

```diff
@@ -1,16 +1,78 @@
-import React from 'react';
+import React from "react";
+import TodoForm from "./components/TodoComponents/TodoForm";
+import TodoList from "./components/TodoComponents/TodoList";
+const tasks = [
+  {
+    task: "Organize Garage",
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: "Bake Cookies",
+    id: 1528817084358,
+    completed: false
+  }
+];
 
 class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      tasks: tasks,
+      input: "test"
```

Comment:
you could initially set `input: ""` here so there's no initial value, and inside your form input add the attribute `placeholder="test"` which will make the input field say "test" but once you start typing in it will go away. take a look at how the placeholder attr. is used on MDN or W3Schools for more examples and how it looks